### ISSUE TITLE
Crafts not being added to the list after first successful screen capture

### ISSUE THREAD

**Issue by Revq** [esge/PoE-HarvestVendor#33]
Since introduction of version 0.4.3 app completely stopped working for me. It works correctly with first screen capture after launching but every next subsequent doesn't add new crafts anymore. App lets me select the screen area and seemingly everything works properly but no new entries show up.
No errors logged. Problem repeats on both ahk and exe version. Present on every version past v0.4.2.
Last working version for me is .ahk 0.4.2 version.

**Comment by esge** [comment]
which crafts ? or could you provide screenshot of your horti station ?

**Comment by Revq** [comment]
Sorry sold them already but I can tell you from my experience that it doesn't matter what crafts are on the screen. The second time I try to screen capture it instantly stops parsing. Ill try to record small demo for you.

**Comment by esge** [comment]
is something written in temp.txt after the failed attempt ?

**Comment by Revq** [comment]
Temp.txt stays completly empty. Notepad++ detects that file was changed (asking for reload) tho.

**Comment by Revq** [comment]
Ok so as you suggested checking this file I decided to run the exe as an Admin and completely remove temp.txt while its running. It now works properly. 
Just as a note: Files were located outside of system partition.
I'll try to replicate the issue again with clean setup.

**Comment by Revq** [comment]
Ok, so in the proccess of me figuring out if it's an issue with permissions on my part (Which btw should not happen) it fixed itself completely and I can't even replicate the issue with new setup. The app seemingly couldn't write to temp.txt which broke everything. I've reseted the file and gave full permission to all groups in the system, which did absolutely nothing, after which I removed the file again and restarted the app. Since then everything works and doesn't want to break even when I'm trying to. I double checked and it doesn't have perms from previous deleted file either. For me issues is fixed (wondering for how long tho) but the cause is still flouting somewhere the in the ether.

I really hate windows.
